Source/Core/DolphinQt/GameList/GameList.cpp

```diff
@@ -675,6 +682,78 @@ void GameList::OpenSaveFolder()
   QDesktopServices::openUrl(url);
 }
 
+void GameList::OpenGCSaveFolder()
+{
+  const auto game = GetSelectedGame();
+  if (!game)
+    return;
+
+  std::string region;
+
+  switch (game->GetRegion())
+  {
+  case DiscIO::Region::NTSC_J:
+    region = "JAP";
+    break;
+  case DiscIO::Region::NTSC_U:
+    region = "USA";
+    break;
+  case DiscIO::Region::PAL:
+    region = "EUR";
+    break;
+  case DiscIO::Region::Unknown:
+    return;
+  }
+
+  bool found = false;
+
+  for (int i = 0; i < 2; i++)
+  {
+    QUrl url;
+    switch (SConfig::GetInstance().m_EXIDevice[i])
+    {
+    case ExpansionInterface::EXIDEVICE_MEMORYCARDFOLDER:
+    {
+      std::string path = StringFromFormat("%s/%s/%s", File::GetUserPath(D_GCUSER_IDX).c_str(),
+                                          region.c_str(), i == 0 ? "Card A" : "Card B");
+
+      std::string override_path = i == 0 ? Config::Get(Config::MAIN_GCI_FOLDER_A_PATH_OVERRIDE) :
+                                           Config::Get(Config::MAIN_GCI_FOLDER_B_PATH_OVERRIDE);
+
+      if (!override_path.empty())
+        path = override_path;
+
+      QDir dir(QString::fromStdString(path));
+
+      if (!dir.entryList({QStringLiteral("01-%1*.gci")
```

Comment:
Kinda the same question about this (seemingly internal detail); do we have some other location that gives us the expected save file name?